src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -865,20 +867,8 @@ public Void visitObjectConstructor(JsoniqParser.ObjectConstructorContext ctx) {
         return null;
     }
 
-    // TODO[EXPRVISITOR]? not supported in Pair constructor
     @Override
     public Void visitPairConstructor(JsoniqParser.PairConstructorContext ctx) {
```

Comment:
This method looks empty now. Can it be completely removed?

Reply:
Apparently, it can because it is not abstract. DONE.